includes/options/views/class-amp-setup-wizard-submenu-page.php

```diff
@@ -104,6 +108,9 @@ public function enqueue_assets( $hook_suffix ) {
 			return;
 		}
 
+		/** @var AmpSlugCustomizationWatcher $amp_slug_customization_watcher */
+		$amp_slug_customization_watcher = Services::get( 'amp_slug_customization_watcher' );
```

Comment:
This should be injected instead, provided that `AMP_Setup_Wizard_Submenu_Page` is instantiated via the injector.

Reply:
〃